src/containers/ConnectorManagement.jsx

```diff
@@ -167,8 +169,8 @@ export default class ConnectorManagement extends Component {
     // if wrong connector oauth
     if (eventData.origin !== `${this.props.params.connectorSlug}_oauth`) return
     // get account id from localStorage event and remove the listener
-    window.removeEventListener('storage', this.terminateOAuth.bind(this))
     const accountID = eventData.key
+    window.removeEventListener('storage', this.terminateOAuth)
```

Comment:
I feel that all this stuff with `window.addEventListener` and things should be in our account lib. What do you think ? Is there something that disallow us to do so ?